### PR TITLE
fix shrike healing & crit healing and properly addresses regen delays

### PR DESCRIPTION
**Pull request by Arturlang**
## About The Pull Request
Didn't realize heal_wounds was used in other places, this gives it a default arg of expected seco

## Why It's Good For The Game
Fixes bad bugs and hopefully makes the revert PR not needed

## Changelog

:cl:
fix: shrike healing and crit healing should now work
add: regen delays now properly are handled by seconds_per_tick
/:cl:

### CODE REVIEW

**Comment by Arturlang** [comment]
The previous PR really should not have been fullmerged, but TM'd instead, but it's my fault since i didn't ask for it

File: code/modules/mob/living/carbon/xenomorph/life.dm [hunk 1]
```diff
@@ -84,22 +84,23 @@
 
 /mob/living/carbon/xenomorph/proc/handle_critical_health_updates(seconds_per_tick)
 	if(loc_weeds_type)
-		heal_wounds(XENO_RESTING_HEAL, seconds_per_tick)
+		heal_wounds(XENO_RESTING_HEAL, seconds_per_tick = seconds_per_tick) // healing in critical while on weeds ignores scaling
```
Comment (Lumipharon) [inline code/modules/mob/living/carbon/xenomorph/life.dm:87]: uh, why this change

**Review by Lumipharon — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/life.dm (hunk 1 above)
Comment (Arturlang) [reply code/modules/mob/living/carbon/xenomorph/life.dm:87]: to just absolutely make sure that it's very obvious what arg this is touching, preventing mistakes, if you don't want it i can remove it

Comment (Lumipharon) [reply code/modules/mob/living/carbon/xenomorph/life.dm:87]: you can already see what its doing because its an arg, literally jut hover over and it will say.

Comment (Arturlang) [reply code/modules/mob/living/carbon/xenomorph/life.dm:87]: changed it to positional args as it seemed what was seemingly requested.

**Review by Lumipharon — APPROVED** [review]
